src/magellan_firmware/firmware/magellan_controller/config.h

```diff
@@ -26,19 +26,23 @@ extern RobotState currentState;
 #define SERVO_PWM 4
 
 // Steering trim
-#define STEERING_OFFSET -0.2
+#define STEERING_OFFSET 0
 
 #define THROTTLE_MIN 0.1
 #define STEERING_MIN 0.05
 
-#define LEFT_ENCODER 20
+#define ENCODER_A 6
+#define ENCODER_B 7
+#define ENCODER_C 8
+
 #define RUN_BUTTON 21
-#define RIGHT_ENCODER 22
 
+
+// TODO: Measure and change vars accordingly
 // Distance from front wheel to back wheel
-const constexpr double kTrackLength = 0.3175;
+const constexpr double kTrackLength = 0.39; // 39 cm
```

Comment:
You can generally assume all ROS measurements are in SI units: https://www.ros.org/reps/rep-0103.html